app/serializers/court_case_serializer.rb

```diff
@@ -0,0 +1,41 @@
+class CourtCaseSerializer < ActiveModel::Serializer
+  attributes :nomis_case_id, :nomis_case_status, :nomis_case_start_date,
+             :nomis_case_type, :nomis_case_number
+
+  belongs_to :location, serializer: LocationSerializer
+
+  def nomis_case_id
+    object.id
```

Comment:
Same thing here with the inconsistent use of `id` - amending this to just `id` means it's more compatible with JSONAPI spec, right?

Reply:
I am using the `object.case_id` now.